metacat-main/src/test/groovy/com/netflix/metacat/main/services/MetacatInitializationServiceSpec.groovy

```diff
@@ -47,6 +47,7 @@ class MetacatInitializationServiceSpec extends Specification {
             threadServiceManager,
             thriftService
         )
+        initializationService.start(null)
```

Comment:
Null? What if you ever do something with the event in the Event listener. Right now you're serializing it as a string but if you ever change it to do something like `event.getSource()` this test is going to barf.

Reply:
At present, we do not. Once we do, the test will raise a flag by failing.